.gitignore

```diff
@@ -55,4 +55,5 @@ src/main/webapp/node_modules
 ### MISC ###
 /.theia/launch.json
 *.swp
-/src/main/resources/application-local.properties
\ No newline at end of file
+/src/main/resources/application-local.properties
```

Comment:
can you add `gradle.properties` to the gitignore since that way developers won't accidentally commit their local changes to it.

Reply:
added.